src/engine/uci_engine.cpp

```diff
@@ -96,10 +96,12 @@ void UciEngine::writeLog() const {
 
 std::string UciEngine::lastInfoLine() const {
     // iterate backwards over the output and save the first line
-    // that contains "info depth" and score
+    // that contains "info depth" and score and multipv 1
```

Comment:
well this is wrong? it doesnt have to contain multipv 1..

Reply:
fixed 👍